trigger_attestation.md

```diff
@@ -0,0 +1,448 @@
+minimal.
+
+### Why deviate from the Private Click Measurement approach?
+
+Private Click Measurement (PCM) supports a [similar
+design](https://webkit.org/blog/12566/private-click-measurement-conversion-fraud-prevention-and-replacement-for-tracking-pixels),
+where source and trigger registration support adding an optional nonce, which is
+signed via an interaction with the server. For the privacy reasons [described
+above](#why-doesnt-this-support-issuance-at-source-registration-time), we can’t
+align with PCM on source signing.
+
+For trigger signing, our approaches are broadly compatible, except that
+
+*   PCM does all configuration via query parameters in a (blocked) redirect and
+    we do everything in headers. This is necessary for us because we support
+    additional registrations with real redirects.
+*   PCM puts the resulting unlinkable token in the body of a report, while we
+    propose to do it in a separate header. This is mostly to align with our
+    header issuance protocol to keep things consistent.
+*   PCM uses
+    [RSABSSA](https://www.ietf.org/archive/id/draft-irtf-cfrg-rsa-blind-signatures-01.html)
+    while this proposal uses [Private State
+    Tokens](https://github.com/WICG/trust-token-api) / [Privacy
+    Pass](https://www.ietf.org/archive/id/draft-ietf-privacypass-architecture-06.html)
+    for underlying crypto operations which sits one layer higher in the stack.
+    Notably, Privacy Pass internally uses RSABSSA. Integrating with Private
+    State Tokens allows us to re-use its key commitment infrastructure.
+
+Other than that the designs are very similar.
+
+### Could we just tag reports with a trigger_id instead of using anonymous tokens?
+
+In principle, we could abandon the token-based approach and tag every report
+with an unencrypted high-entropy identifier that is configured at trigger time.
+This would uniquely tie every encrypted report with the trigger context that
+generated it. Because the ids are high entropy (and thus difficult to guess),
+they could serve the purpose of the anonymous tokens and additionally allow for
+offline filtering of reports (i.e. deferring the IVT decision until after
+trigger registration).
+
+This will allow a reporting origin to partition encrypted reports on a
+per-advertiser-user basis, which means that in order to make this idea privacy
+neutral, we’d need to fully protect the counts of encrypted reports e.g. by
+unconditionally sending aggregatable reports on every trigger registration (even
+when there is no corresponding source), ensuring that the encrypted payloads
+reveal no cross-site information. See issue #439 for more details. We are not
+proceeding with this option currently due to performance concerns (bandwidth,
+compute) with sending lots of null reports.
+
+Note that this approach would also mean we could likely remove some/all of the
+delay in aggregatable reports. However, this approach requires…
```

Comment:
Should we leave a TODO to consider documenting this out more if this is intentionally left unspecified?